app/controllers/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class UsersController < ApplicationController
+  def new
+  end
+
+  def show
+    @user = User.find(params[:id])
+  end
+
+  def create
+    user = User.create(user_params)
+    redirect_to "/users/#{user.id}"
```

Comment:
Looks great. Maybe in the future we could add a flash message to confirm that a user was created.